.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -98,6 +98,8 @@ body:
         Which version of Apache DolphinScheduler are you running? We only accept bugs report from the LTS projects.
       options:
         - dev
+        - 2.0.3
```

Comment:
it seem we do not release 2.0.3. it there any reason yiu want to add it?

Reply:
It is a bug test requirement before 2.0.3 release.